internal/nets/paramstores/redis_test.go

```diff
@@ -0,0 +1,95 @@
+package paramstores
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/qvantel/nerd/internal/config"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+var testRedisStore NetParamStore
+
+func getTestStore(url string) (NetParamStore, error) {
+	var storeParams map[string]interface{}
+	json.Unmarshal([]byte(`{"URL": "`+url+`"}`), &storeParams)
+	conf := config.Config{
+		ML: config.MLParams{
+			StoreType:   config.RedisParamStore,
+			StoreParams: storeParams,
+		},
+	}
+	return New(conf)
+}
+
+func startRedis(ctx context.Context) (redis testcontainers.Container, url string, err error) {
+	req := testcontainers.ContainerRequest{
+		Image:        "redis:6.0.9-alpine",
```

Comment:
There is a newer redis image available, but maybe you're using this version for a reason :)

Reply:
Good catch, I wasn't forcing the alpine version either, I've updated it and pushed it